code/game/jobs/squads.dm

```diff
@@ -78,6 +78,12 @@
 	radio_freq = DELTA_FREQ
 
 
+/datum/squad/proc/get_all_members()
+	return marines_list
+
+/datum/squad/proc/get_total_members()
```

Comment:
How about turning these into macros, since they are not redefined anywhere?
`#define SQUAD_TOTAL_MEMBERS(squad) length(squad.marines_list)`
``#define SQUAD_ALL_MEMBERS(squad) squad.marines_list``

Reply:
see #1253 for why not